http/headers/Expect-CT.json

```
@@ -41,7 +39,7 @@
           "status": {
             "experimental": false,
             "standard_track": true,
-            "deprecated": false
+            "deprecated": true
```

We shouldn't change `deprecated` to true if a _browser_ deprecates the feature, but rather if the _spec_ deprecates it.

```suggestion
            "deprecated": false
```

---

Thanks for the clarification, I updated this!